Makefile

```diff
@@ -131,9 +131,15 @@ rpm: $(TARFILE) $(SPEC)
 	rm -r "`pwd`/rpmbuild"
 	rm -r "`pwd`/output" "`pwd`/build"
 
+NETWORK=
+# fedora can be built offline; the rest does not pull podman containers in image-create yet
+ifeq ($(filter fedora,$(TEST_OS)),)
```

Comment:
Don't you want `ifneq` here? I.e. add `--no-network` for Fedora images, not "everything but Fedora images"?

Reply:
I've found https://www.gnu.org/software/make/manual/make.html#Testing-Flags and man this is confusing.